templates/activity.mustache

```diff
@@ -64,9 +64,40 @@
     <!-- Only display mastery for practice activities. -->
     {{#sectionid}}
     <span class="emphasis-bold answer">Chapter Mastery (<span id="chapter-mastery-title"></span>):</span><br>
-    <progress class="progress" id="chapter-mastery" value="0" max="100" data-text="0%"></progress><br>
+        <svg style="display:none;">
```

Comment:
Should this be hidden?

Reply:
Hi @HAASLEWER yes it should be otherwise the format does not look right:)